templates/field/field--node--oe-consultation-documents--full.html.twig

```diff
@@ -7,7 +7,7 @@
  */
 #}
 {% for item in items %}
-  <div class="ecl-u-mt-m">
+  <div class="ecl-u-mb-l">
```

Comment:
Why the increase in the margin?

Reply:
To make it consistent with Person. See image on the https://github.com/openeuropa/oe_theme/pull/742/files/d1c1a98b608ef027839f46fc507c64b9e6d95782#diff-c3ac261fd2860e4533d87febebc2ba8e7bcf9e0935ad90d7fb2e30ff0df3cda0